src/views/recordTable.component.tsx

```diff
@@ -59,14 +59,27 @@ const RecordTable = React.memo(
               (channel) => channel.systemName === keys[i]
             );
             const newColumn: Column = {
-              Header: channelInfo
-                ? channelInfo.userFriendlyName
+              Header: () => {
+                const headerName = channelInfo
                   ? channelInfo.userFriendlyName
-                  : channelInfo.systemName
-                : keys[i], // Provide an actual header here when we have it
+                    ? channelInfo.userFriendlyName
+                    : channelInfo.systemName
+                  : keys[i];
+                // Provide an actual header here when we have it
+                // TODO: do we need to split on things other than underscore?
```

Comment:
Couldn't see anything in the requirements or design docs that currently indicates so

Reply:
I'll keep this TODO here, as we currently don't know what the format of the channel names are